File: airesources/CSharp/hlt/Player.cs
using System.Collections.Generic;

public class Player {

    public List<Ship> Ships
    {
        get
        {
            return ships;
        }

        set
        {
            ships = value;
        }
    }

    public int Id
    {
        get
        {
            return id;
        }
    }

    private List<Ship> ships;
    private int id;

    public Player(int id){
        this.id = id;
        ships = new List<Ship>();
    }
}

File: airesources/Java/hlt/Player.java
package hlt;

import java.util.Collections;
import java.util.Map;
import java.util.TreeMap;

public class Player {

    private final Map<Integer, Ship> ships;
    private final int id;

    public Player(final int id, Map<Integer, Ship> ships) {
        this.id = id;
        this.ships = Collections.unmodifiableMap(ships);
    }

    public Map<Integer, Ship> getShips() {
        return ships;
    }

    public Ship getShip(final int entityId) {
        return ships.get(entityId);
    }

    public int getId() {
        return id;
    }
}
